.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-{} AND/OR ISSUE: https://github.com/aerogear/standards/issues/{}) 
+
+## What
```

Comment:
I also find the same with `What` and `How` these end up being very similar for me so I often delete one or the other.

Reply:
I agree.